roles/nameserver/README.rst

```diff
@@ -0,0 +1,170 @@
+nameserver
+==========
+
+This role is used to set up and configure a very basic **internal** BIND DNS master server.
+
+This role has only been tested on CentOS 7.2 using BIND9.
+
+It does the following:
+
+- Installs and updates necessary packages
+- Enables and configures firewalld
+- Manages named.conf and BIND daemon config
+- Manages forward and reverse DNS records
+
+Prerequisites
++++++++++++++
+
+- CentOS 7.2
+
+Variables
++++++++++
+Most variables are defined in ``roles/nameserver/defaults/main.yml`` and values are chosen to support our Sepia_ lab.  They can be overridden in the ``secrets`` repo.
+
+A list of packages to install that is specific to the role.  The list is defined in ``roles/nameserver/vars/packages.yml``::
```

Comment:
I'm confused by this style of "define the variable, and then show it"; it may be what other doc files have done, but I think it'd be way better to say 

packages: []

A list of packages to install....